src/.goreleaser.yml

```
@@ -64,7 +64,7 @@ archives:
   - name_template: '{{ .ProjectName }}-{{ .Os }}-{{ .Arch }}'
     format_overrides:
       - goos: windows
-        format: zip
+        format: [ 'zip' ]
```

The deprecation was [changing the key from format to formats](https://goreleaser.com/deprecations/#archivesformat). Apparently single string should still be fine?

> It will still accept a single string, e.g.: formats: zip. In most cases you can simply rename the property to formats.

```suggestion
        formats: zip
```